cmd/stellar-rpc/internal/ingest/service_test.go

```diff
@@ -98,12 +98,11 @@ func createTestLedger(t *testing.T) xdr.LedgerCloseMeta {
 	return xdr.LedgerCloseMeta{
 		V: 1,
 		V1: &xdr.LedgerCloseMetaV1{
-			LedgerHeader:                   createLedgerHeader(),
-			TxSet:                          createTransactionSet(),
-			TxProcessing:                   createTransactionProcessing(t),
-			UpgradesProcessing:             []xdr.UpgradeEntryMeta{},
-			EvictedTemporaryLedgerKeys:     []xdr.LedgerKey{createEvictedTempLedgerKey()},
-			EvictedPersistentLedgerEntries: []xdr.LedgerEntry{createEvictedPersistentLedgerEntry()},
+			LedgerHeader:       createLedgerHeader(),
+			TxSet:              createTransactionSet(),
+			TxProcessing:       createTransactionProcessing(t),
+			UpgradesProcessing: []xdr.UpgradeEntryMeta{},
+			EvictedKeys:        []xdr.LedgerKey{createEvictedTempLedgerKey()},
```

Comment:
We should still add a `createEvictedPersistentLedgerEntry()`, right?

Reply:
I don't think so, since the field is not used anymore

Reply:
I thought the semantics have changed: that is, `EvictedKeys` will have both temporary and persistent entries now.

Reply:
I don't think the entries were verified in the test.